src/modules/Builder/features/PerformanceTasks/Flanker/RoundSettings/BlockSequences/BlockSequencesContent/BlockSequencesContent.tsx

```diff
@@ -27,7 +27,7 @@ import {
 } from './BlockSequencesContent.utils';
 import { UploadedTable } from './BlockSequencesContent.types';
 
-export const BlockSequencesContent = ({ isPracticeRound }: IsPracticeRoundType) => {
+export const BlockSequencesContent = ({ isPracticeRound, onError }: IsPracticeRoundType) => {
```

Comment:
pls create it's own type instead of IsPracticeRoundType

Reply:
done